src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Definition.cs

```diff
@@ -14,5 +14,22 @@ public static readonly
             RequestType<TextDocumentPosition, Location[]> Type =
             RequestType<TextDocumentPosition, Location[]>.Create("textDocument/definition");
     }
+
+    /// <summary>
+    /// Parameters sent back with a definition event
+    /// </summary>
+    public class DefinitionSentParams
+    {
+    }
+
+    /// <summary>
+    /// Event sent when the language service sent the definition
+    /// </summary>
+    public class DefinitionSentNotification
```

Comment:
I was thinking more of a generic SendTelemetryNotification that asks for telemetry to be published? This would make it much cheaper to add new telemetry from the Service side since we can add them there and not update the VSCode-MSSQL side at all. What do you think?